src/nidigital/examples/nidigital_ppmu_source_and_measure.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import pytest
```

Comment:
I don't think you need `pytest`

Reply:
For test, I am using pytest to make sure an exception is raised when the user passes on argument to measure current without sourcing voltage or current.

Reply:
hmm.. I suppose by design we're including `test_` methods in the same module as the examples. So have to `import pytest` to use anything provided by `pytest`.

Maybe we should move the example test methods into a separate module (outside the scope of this PR). @marcoskirsch, what do you think?

Reply:
If we don't want to depend on pytest in our examples, I can use try..catch + assert to achieve the same goal.

Reply:
@sbethur @marcoskirsch Should we remove the pytest and replace it with a try..catch + assert?

Reply:
hmm.. not ideal, but I think it's better given we can get rid of the dependency.

Reply:
>Maybe we should move the example test methods into a separate module (outside the scope of this PR). @marcoskirsch, what do you think?

I think this makes sense.

>@sbethur @marcoskirsch Should we remove the pytest dependency and replace it with a try..catch + assert?

I don't like this option. By using pytest the intent is clear.